localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -2697,7 +2703,7 @@ msgstr "Tarea de envío de impresión fallida."
 msgid "Service Unavailable"
 msgstr "Servicio No Disponible"
 
-msgid "Unknown Error."
+msgid "Unkown Error."
```

Comment:
Should stay as Unknown.

Reply:
@SoftFever should correct it in original PO.